hummingbird/ml/operator_converters/__init__.py

```diff
@@ -31,4 +32,5 @@
 __pdoc__ = {}
 __pdoc__["hummingbird.operator_converters._gbdt_commons"] = True
 __pdoc__["hummingbird.operator_converters._tree_commons"] = True
+__pdoc__["hummingbird.operator_converters._linear_implementations"] = True
```

Comment:
I think this should be `_normalizer_implementations`,

Reply:
whoa.  how was this working?   (the bug was from a cherrypick of my other PR, but I"m surprised this worked at alll)

Reply:
ah yes.  just pdoc. :-P right.